tests/conftest.py

```diff
@@ -31,30 +31,44 @@
 from __future__ import absolute_import, print_function, unicode_literals, division
 from contextlib import closing
 
+import kirin
 from kirin import app, db
```

Comment:
```suggestion
from kirin import app, db, redis
```
To avoid the previous line because only kirin.redis is used